src/routes/health.js

```diff
@@ -0,0 +1,9 @@
+import { StatusCodes } from 'http-status-codes'
+
+export const health = {
+  method: 'GET',
+  path: '/health',
+  handler: function (_request, h) {
+    return h.response('ok').code(StatusCodes.OK)
```

Comment:
No need for the `.code` extension here as will return `200` be default.

Also recommend we make the structure and response consistent with the backend health check.

`handler: (_request, h) => h.response({ message: 'success' })`